core/schema/src/main/liquibase/30.0.0/changelog.xml

```diff
@@ -27,19 +27,17 @@
             <column name="interface_id" type="integer">
```

Comment:
Could this field be renamed `ipinterface_id` to make it clear it's a foreign key to the `ipinterface` table?